crates/cheatnet/src/forking/cache.rs

```diff
@@ -79,35 +80,19 @@ impl ToString for ForkCacheContent {
 pub struct ForkCache {
     fork_cache_content: ForkCacheContent,
     cache_file: Option<String>,
-    block_id: BlockId,
 }
 
 impl Drop for ForkCache {
     fn drop(&mut self) {
-        if !matches!(self.block_id, BlockId::Tag(_)) {
-            self.save();
-        }
-    }
-}
-
-fn block_id_to_string(block_id: BlockId) -> String {
-    match block_id {
-        BlockId::Hash(x) => Felt252::from_(x).to_str_radix(16),
-        BlockId::Number(x) => x.to_string(),
-        BlockId::Tag(x) => match x {
-            BlockTag::Latest => "latest".to_string(),
-            BlockTag::Pending => unreachable!(),
-        },
+        self.save();
```

Comment:
I wonder if that won't cause the cache to grow very fast in case someone uses `latest` in their tests. We'd be caching a new block every couple runs basically.

Reply:
That's true, but it that really an issue? Cache can always be cleared

Reply:
But we don't have any automatic mechanism of doing so. I don't know how large can a single block cache be, but it should be investigated if an issue like this concerns us.

cc @war-in

Reply:
Meh i think the cache files are extremely small and shouldn't really worry us

Reply:
And tbh latest never got there before anyways, we always got the block number before the test, so we already always cached it xD

Reply:
In the beginning, it was there. Some changes were made and we forgor about it